tests/subsys/logging/log_api/boards/qemu_leon3.conf

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# QEMU crashes during log2_api_immediate_printk_cpp.
+# Workaround it by making the main stack larger.
+# See #46056.
+CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
For future reference, the apparent cause of this failure was investigated here: https://discord.com/channels/720317445772017664/733037890514321419/979782761298661386.

> hey I've been bisecting that one, very interesting one, starts to fail once the version string patch goes over 99, as in v3.1.0-rc2-100-g39c8ba32ff3b fails, v3.1.0-rc2-99-g97d989910ebe. Does it for every tag, changing the abbrev flag in git describe --abbrev=12 also makes it pass of fail, looks like it breaks depending on the version string length